Library/Homebrew/dev-cmd/unbottled.rb

```diff
@@ -243,4 +256,53 @@ def output_unbottled(formulae, deps_hash, noun, hash, any_named_args)
 
     puts "No unbottled dependencies found!"
   end
+
+  def output_lost_bottles
+    # $ git log --patch -G\^\ \+sha256.\*\ sonoma: --since=@\{\'1\ week\ ago\'\}
+    git_log = %w[git log --patch]
+    git_log << "-G^ +sha256.* #{@bottle_tag}:"
+    git_log << "--since=@{'1 week ago'}"
+
+    bottle_tag_sha_regex = /^[+-] +sha256.* #{@bottle_tag}: /
+
+    processed_formulae = Set.new
+    commit = T.let(nil, T.nilable(String))
+    formula = T.let(nil, T.nilable(String))
+    lost_bottles = 0
+
+    CoreTap.instance.path.cd do
+      Utils.safe_popen_read(*git_log) do |io|
+        io.each_line do |line|
+          case line
+          when /^commit [0-9a-f]{40}$/
+            # Example match: `commit 7289b409b96a752540befef1a56b8a818baf1db7`
+            if commit && lost_bottles.positive? && processed_formulae.exclude?(formula)
+              puts "#{commit}: bottle lost for #{formula}"
+            end
+            processed_formulae << formula
+            commit = line.split.last
+            lost_bottles = 0
```

Comment:
```suggestion
            formula = nil
```

Reply:
Handled in https://github.com/Homebrew/brew/pull/16115/commits/83598bb950bdb55b23232f7fe6767852aa856a23